triples-generator/src/main/scala/ch/datascience/triplesgenerator/eventprocessing/triplesgeneration/renkulog/RenkuLogTriplesGenerator.scala

```diff
@@ -82,6 +86,18 @@ private[eventprocessing] class RenkuLogTriplesGenerator private[renkulog] (
       _             <- (git.checkout(commitEvent.commitId, repoDirectory)).toRight
     } yield ()
 
+  private def cleanUpRepository(repoDirectory: Path) = {
+    val gitAttributeFilePath: Path = repoDirectory / gitAttributeFileName
+    file.exists(gitAttributeFilePath).flatMap {
+      case false => IO.unit
+      case true =>
+        for {
```

Comment:
I guess it would be nice if `repoDirectory` is implicit by I think it's not a simple change...